tests/dialects/test_vector.py

```diff
@@ -163,83 +163,134 @@ def test_vector_broadcast_verify_type_matching():
 
 
 def test_vector_fma():
-    i32_lhs_vector_type = VectorType.from_type_and_list(i32, [])
-    i32_rhs_vector_type = VectorType.from_type_and_list(i32, [])
-    i32_acc_vector_type = VectorType.from_type_and_list(i32, [])
-
-    lhs_vector_ssa_value = OpResult(i32_lhs_vector_type, [], [])
-    rhs_vector_ssa_value = OpResult(i32_rhs_vector_type, [], [])
-    acc_vector_ssa_value = OpResult(i32_acc_vector_type, [], [])
```

Comment:
I don't think we want to combine the SSAValues into jsut one. Then the assertions further down don't make much sense anymore.